lua/neo-tree/sources/diagnostics/init.lua

```diff
@@ -94,6 +94,27 @@ M.diagnostics_changed = function()
   )
 end
 
+M.autopreview = function(preview_config)
+  local state = get_state()
+  if state.preview and state.preview.active then
```

Comment:
This was actually changed in the current version of neo-tree, the `preview` object is no longer on the state and is now a global singleton accessed through the preview module. 

There is no way to see if it is active in the current release, but I added a method in main that I will release today.

The current way to do this in `main` is:

```lua
  local preview = require("neo-tree.sources.common.preview")

  if preview.is_active() then
    return
  end
```